calibration/CTDMOG/CGINS-CTDMOG-13425__20150507.csv

```diff
@@ -10,16 +10,16 @@ serial,name,value,notes
 37-13425,CC_i,-0.0001245663,
 37-13425,CC_j,2.811515e-05,
 37-13425,CC_p_range,1450,
-37-13425,CC_pa0,0.1941297,
-37-13425,CC_pa1,0.004950054,
-37-13425,CC_pa2,-2.066723e-11,
-37-13425,CC_ptca0,525111.2,
-37-13425,CC_ptca1,2.700952,
-37-13425,CC_ptca2,-0.04099572,
-37-13425,CC_ptcb0,25.48337,
-37-13425,CC_ptcb1,0.000275,
+37-13425,CC_pa0,2.664023e-01,
+37-13425,CC_pa1,4.568826e-03,
+37-13425,CC_pa2,-2.266358e-11,
+37-13425,CC_ptca0,5.252145e+05,
+37-13425,CC_ptca1,7.199276e+00,
+37-13425,CC_ptca2,-2.007133e-01,
+37-13425,CC_ptcb0,2.507800e+01,
+37-13425,CC_ptcb1,-0.000000e+00,
 37-13425,CC_ptcb2,0,
-37-13425,CC_ptempa0,-72.90904,
-37-13425,CC_ptempa1,0.05137249,
-37-13425,CC_ptempa2,-7.216569e-07,
-37-13425,CC_wbotc,1.8588e-07,
+37-13425,CC_ptempa0,1.293224e+02,
+37-13425,CC_ptempa1,-6.012567e-02,
+37-13425,CC_ptempa2,-1.537901e-06,
+37-13425,CC_wbotc,1.858827e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT